userlevel/click.cc

```diff
@@ -98,7 +98,7 @@ static const Clp_Option options[] = {
     { "simtime", 0, SIMTIME_OPT, Clp_ValDouble, Clp_Optional },
     { "simulation-time", 0, SIMTIME_OPT, Clp_ValDouble, Clp_Optional },
     { "threads", 'j', THREADS_OPT, Clp_ValInt, 0 },
-    { "affinity", 'a', THREADS_AFF_OPT, 0, 0 },
+    { "cpu", 'a', THREADS_AFF_OPT, Clp_ValInt, Clp_Optional },
```

Comment:
Don't like changing the name of this option. Leave `--affinity` around. OK to provide a silent synonym, `--cpu`, in addition,